proto/crosschain/genesis.proto

```diff
@@ -24,4 +24,5 @@ message GenesisState {
 
   repeated LastBlockHeight lastBlockHeightList = 8;
   repeated InTxHashToCctx inTxHashToCctxList = 9 [(gogoproto.nullable) = false];
+  repeated TSS tss_history = 10;
```

Comment:
Any reason to not use `[(gogoproto.nullable) = false]`? I feel like using pointers increase complexity and the risk of getting nil dereference